packaging/amazon-linux-ami/ecs.service

```diff
@@ -16,9 +16,12 @@
 Description=ECS Agent
 Requires=docker.service
 After=docker.service
+After=cloud-final.service
 
 [Service]
 Type=simple
+Restart=on-failure
+RestartSec=10s
```

Comment:
Should we add a `StartLimitIntervalSec` or `StartLimitBurst`? Or just continue to restart forever? (restarting forever matches the behavior of the upstart managed service.

Reply:
There are certainly more options we can start adding, but we really should have ecs-init itself start cooperating and allowing the init system to manage the process to have meaningful effect.

Reply:
sgtm, :boat: :mortar_board: